worlds/dark_souls_3/__init__.py

```diff
@@ -822,7 +810,7 @@ def _add_npc_rules(self) -> None:
         quest.
         """
 
-        ## Greirat
+        # Greirat
```

Comment:
I'd like to keep these double `#`s, since they indicate headers rather than normal comments.